src/main/java/seedu/duke/command/DeleteAllCommand.java

```diff
@@ -6,10 +6,18 @@
 import seedu.duke.task.TaskList;
 import seedu.duke.ui.Ui;
 
+import java.io.IOException;
+
 public class DeleteAllCommand extends DeleteCommand {
     @Override
     public void execute(Ui ui, TaskList taskList, LessonList lessonList, Storage storage) throws DukeException {
         taskList.clearTaskList();
         lessonList.clearLessonList();
+        ui.printDeletedAll();
+        try {
+            storage.saveData(taskList, lessonList);
+        } catch (IOException e) {
+            throw new DukeException(e.toString());
+        }

```

Comment:
Actually we can simplify this to a single line.

```java
storage.saveData(taskList);
```

Since the `execute` method already `throws` the exception, don't need to use `try-catch`. Also, masking an `IOException` into a `DukeException` is probably not a good practice.